app/assets/stylesheets/components/_back-to-top.scss

```diff
@@ -1,13 +1,11 @@
-.back-to-content {
+.app-c-back-to-top {
   @include core-16;
   display: inline-block;
   margin-bottom: $gutter-half;
 
-  .icon {
-    float: left;
-    margin-right: .3em;
-    width: .8em;
-    height: .968em;
+  // To compensate for the -$gutter-half from @include sidebar-with-body
+  .detailed-guide & {
```

Comment:
This doesn't seem isolated 🤔

Reply:
Could this be achieved with a wrapping element?

Reply:
This has been resolved.